open_street/osm_geometry.py

```diff
@@ -0,0 +1,76 @@
+import osmium
+from osmium.geom import GeoJSONFactory
+from typing import Any
+
+
+class GeometryRoadHandler(osmium.SimpleHandler):
+    def __init__(self) -> None:
+        osmium.SimpleHandler.__init__(self)
+        self.geometryDictionary: dict[str, str] = {}
+
+    def way(self, w: Any) -> None:
+        """This function gets called upon .apply_file for every road (OSM Way) that is in the file that apply_file is called with.
```

Comment:
Synes det her lyder lidt convoluted. Idea:
"Way() looks for all ways with the tag "highway" and adds their osm id and linestring to the geometryDictionary. This function is used by apply_file(), which is method inherited by osmium.SimpleHandler - the handler will look for a function called way() and apply the function on each OSM way (road) in a given .pbf file."

Reply:
Nice! har rettet (mini) i det :)